mpas_analysis/obs/observational_datasets.xml

```diff
@@ -0,0 +1,947 @@
+      to select a final MLD estimate. Results are also presented for MLDs
+      calculated using de Boyer Montegut et al.'s (2004) threshold values.
+    </description>
+    <source>
+      [UCSD Mixed Layer Website](http://mixedlayer.ucsd.edu/)
+    </source>
+    <releasePolicy>
+      [Acknowledgment:](http://mixedlayer.ucsd.edu/) If you use this data,
+      please cite it as: Holte, J., L. D. Talley, J. Gilson, and D. Roemmich
+      (2017), An Argo mixed layer climatology and database, Geophys. Res.
+      Lett., 44, 5618–5626, doi:10.1002/2017GL073426.
+    </releasePolicy>
+    <references>
+      - [Holte et al. (2017)](http://onlinelibrary.wiley.com/doi/10.1002/2017GL073426/full)
+      - [Holte and Talley (2009)](http://journals.ametsoc.org/doi/abs/10.1175/2009JTECHO543.1)
+      - [de Boyer Montegut et al. (2004)](https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/2004JC002378)
+    </references>
+    <bibtex>
+      @article{Holte2017,
+      author = {James Holte and Lynne D. Talley and John Gilson and Dean Roemmich},
+      title = {An Argo mixed layer climatology and database},
+      journal = {Geophysical Research Letters},
+      year = {2017},
+      volume = {44},
+      number = {11},
+      pages = {5618-5626},
+      keywords = {Argo, mixed layer, climatology, global},
+      doi = {10.1002/2017GL073426},
+      url = {https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1002/2017GL073426}
+      }
+
+      @article{Holte2009,
+      author = { James  Holte  and  Lynne  Talley },
+      title = {A New Algorithm for Finding Mixed Layer Depths with Applications to Argo Data and Subantarctic Mode Water Formation},
+      journal = {Journal of Atmospheric and Oceanic Technology},
+      volume = {26},
+      number = {9},
+      pages = {1920-1939},
+      year = {2009},
+      doi = {10.1175/2009JTECHO543.1},
+      URL = {https://doi.org/10.1175/2009JTECHO543.1}
+      }
+
+      @article{deBoyerMontegut2004,
+      author = {Cl\'{e}ment {de Boyer Mont\'{e}gut} and Gurvan Madec and Albert S. Fischer and Alban Lazar and Daniele Iudicone},
+      title = {Mixed layer depth over the global ocean: An examination of profile data and a profile‐based climatology},
+      journal = {Journal of Geophysical Research: Oceans},
+      volume = {109},
+      number = {C12},
+      pages = {},
+      year = {2004},
+      keywords = {mixed layer, mixed layer depth criterion, density compensation},
+      doi = {10.1029/2004JC002378},
+      url = {https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/2004JC002378}
+      }
+    </bibtex>
+    <dataUrls>
+      - http://mixedlayer.ucsd.edu/data/Argo_mixedlayers_monthlyclim_03192017.nc
+    </dataUrls>
+    <preprocessing>
+    </preprocessing>
+    <tasks>
+      - climatologyMapMLD
+    </tasks>
+  </observation>
+
+  <observation>
+    <name>
+      Meridional Heat Transport (MHT)
+    </name>
+    <component>
+      ocean
+    </component>
+    <description>
+     (missing)
```

Comment:
@maltrud, can you help fill in all the missing info about the Trenberth and Caron MHT data set?